src/gpodder/config.py

```diff
@@ -228,6 +230,7 @@
     'auto_remove_unfinished_episodes': 'auto.cleanup.unfinished',
     'auto_remove_unplayed_episodes': 'auto.cleanup.unplayed',
     'max_episodes_per_feed': 'limit.episodes',
+    'check_connection': 'check_connection',
```

Comment:
is this really needed?

Reply:
It was needed for a checkbox I added a week or two ago but apparently removing it seems to work for this one. Why do the other checkboxes need to be there?

Reply:
It's to map the old short 'max_episodes_per_feed' key to the 'episodes' key in the new 'limit' object.